scripts/unix-bundler.sh

```diff
@@ -42,30 +42,38 @@ release_name=$(echo $jar_name | sed 's/.jar//')-$arch
 
 # make release dir
 mkdir $release_name
-
-echo "Dealing with jre..."
-if [ ! -f $jre ]; then
-  curl -L $jre_url -o $jre
-fi
-tar xvf $jre
-mv $jre_dir $release_name/jre
-
-echo "Dealing with electron"
-if [ ! -f $electron ]; then
-  curl -L "https://github.com/electron/electron/releases/download/$electron_version/$electron" -o $electron
+if [ $os = linux ] || [ $os = macOS ]; then
+  echo "Dealing with jre..."
+  if [ ! -f $jre ]; then
+    curl -L $jre_url -o $jre
+  fi
+  tar xvf $jre
+  mv $jre_dir $release_name/jre
+
+  echo "Dealing with electron"
+  if [ ! -f $electron ]; then
+    curl -L "https://github.com/electron/electron/releases/download/$electron_version/$electron" -o $electron
+  fi
+  unzip $electron -d $release_name/electron
```

Comment:
At these point it seems to me that moving the deb builder script to a separate file is appropriate, it's just cluttering the previous order we had

Reply:
What should I name it? `debian-bundler.sh`? `debian-packager.sh`?